### ISSUE TITLE
Tables seem to be used for layout but contain captions

### ISSUE THREAD

**Issue by SavvasStephanides** [open-sdg/open-sdg#739]
**Please complete the following information** 
Site repository URL: github.com/open-sdg/open-sdg
Open SDG version: 1.0.0

**Describe the bug**

Pa11y seems to recognise the data table in the Indicator page as a layout table, reporting the existence of a caption element which should exist in a layout table. This seems to be a problem only when the site is mono-lingual (eg. only available in English)

**To Reproduce**

1. Run a Pa11y test on any indicator page on a mono-lingual site
2. The error should appear in the report.

**Expected behavior**

Since the table is populated with data, it shouldn't be read as a layout table and therefore the caption element should be accepted (see screenshot)

**Screenshots**

![image](https://user-images.githubusercontent.com/478770/84403448-a1550280-abfd-11ea-9f8e-526bd592719f.png)

**Additional context**
Tested on desktop and mobile, with default and high contrast

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning could this be closed as no issue remaining from DAC?

**Comment by brockfanning** [comment]
Works for me!
